hoomd/state.py

```diff
@@ -616,3 +650,33 @@ def thermalize_particle_momenta(self, filter, kT):
         self._simulation._warn_if_seed_unset()
         group = self._get_group(filter)
         group.thermalizeParticleMomenta(kT, self._simulation.timestep)
+
+    @property
+    def domain_decomposition_split_fractions(self):
+        """tuple(list[int], list[int], list[int]): Box fractions of the domain \
```

Comment:
These are not cumulative fractions. The cumulative fraction data structure starts at 0.0 and ends with 1.0. The lists returned here are the box fractional positions of the split planes. For example, a (1,2,4) decomposition would lead to `domain_decomposition_split_fractions == ([], [0.5], [0.25, 0.5, 0.75])`. 

`Box` does not have a method to convert fractional to positional coordinates, nor have we introduced a math notation for the box lattice vectors. We will do so in #1078 and can document that here when we introduce such a notation.